pages/groups.tsx

```diff
@@ -2,8 +2,55 @@ import Header from '../components/Header';
 import { IGroup } from '../components/icons';
 import NewButton from '../components/NewButton';
 import Banner from '../components/Banner';
+import Dropdown, { Options } from '../components/Dropdown';
+import { useState } from 'react';
 
 export default function Groups(): JSX.Element {
+  const optionsGenre = [
+    {
+      label: 'Rock',
```

Comment:
In the DB, it''s name and not label

Reply:
same shit, default select have `label` as a naming convention